database/setup-db.sh

```diff
@@ -171,7 +171,7 @@ create_other() {
 
 grant() {
   # psql -c "ALTER DATABASE \"$RCT_DB_NAME\" OWNER TO \"$RCT_DB_USERNAME\""
-  # psql -c "GRANT ALL PRIVILEGES ON DATABASE \"$RCT_DB_NAME\" TO \"$RCT_DB_USERNAME\""
+  psql -c "GRANT ALL PRIVILEGES ON DATABASE \"$RCT_DB_NAME\" TO \"$RCT_DB_USERNAME\""
```

Comment:
This seems to be really dangerous. You should not have to comment out lines of code to change configuration, environment variables are done for that.

Reply:
ok, I forgot about that